AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -2871,6 +2871,8 @@ jianshu.com#$#body { overflow: auto !important; }
 jianshu.com##.dt-open-bg[data-scene="leftBtn"]
 jianshu.com##.note-comment-above-ad-wrap
 instiz.net#$?#.backlogo { background-image: none !important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/133556
+oshiete.goo.ne.jp###connection_area > div.clearfix:nth-child(2)
```

Comment:
The rule below is enough. Btw, this is not the annoyance. Please move the rule below to the AdGuard Japanese - specific.txt.
```
oshiete.goo.ne.jp##div[style^="margin:20px"][class="clearfix"]
```

Reply:
this rule is not said will block a lot of content on this web

Reply:
Next time please also full fill-in the pull request.

Reply:
> Next time please also full fill-in the pull request.

ok but the rule you suggested will block website content

Reply:
> this rule is not said will block a lot of content on this web

What did you mean? It's better to get rid of using `:nth-child()`, especially when you can create rule without it.

Reply:
> ok but the rule you suggested will block website content

Could you provide an example?

Reply:
> > this rule is not said will block a lot of content on this web
> 
> What did you mean? It's better to get rid of using `:nth-child()`, especially when you can create rule without it.

i think should add id based rule like this
```
oshiete.goo.ne.jp###connection_area > div.clearfix
```

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/133877#discussion_r1012543354 this rule is also acceptable, but I'm looking forward for your [example](https://github.com/AdguardTeam/AdguardFilters/pull/133877#discussion_r1012542525).

Reply:
here is an element i just found
![image](https://user-images.githubusercontent.com/45375496/199664233-7e468cb0-4218-4d23-ba92-7a03f41978fc.png)

Reply:
Well, have you checked the selector? It does not match this element.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/199664715-74030838-3601-4c81-9f0c-dbbba19edfeb.png)

</details><br/>

Reply:
> Chà, bạn đã kiểm tra bộ chọn chưa? Nó không phù hợp với yếu tố này.
> 
> Ảnh chụp màn hình:
> ![hình ảnh](https://user-images.githubusercontent.com/33488229/199664715-74030838-3601-4c81-9f0c-dbbba19edfeb.png)



> Well, have you checked the selector? It does not match this element.
> 
> Screenshot:

your selector can solve the problem but it caused problem for me it blocked 1 important element on the web page.

and I'm really not sure about a selector containing `style` that looks like it could change at any time.

Reply:
Could you please show it via screenshot like I did? Does the selector match the useful element on your end?

> I'm really not sure about a selector containing style that looks like it could change at any time

We have similar rules. If the owner wants to make changes, he will do it regardless of which rule you add.

Reply:
you can see here your rule will block the website's search bar

![image](https://user-images.githubusercontent.com/45375496/199666874-76ed9078-f28a-4344-8cbc-f2825aeb93ea.png)

Reply:
I don't see that the rule matches the element.
In the search you should paste only the selector (not entire rule) and show me the match counter as shown in my screenshot above.

Reply:
> I don't see that the rule matches the element. In the search you should paste only the selector (not entire rule) and show me the match counter as shown in my screenshot above.

oh sorry i double checked your rule don't block important element

but I'm really not sure about using a selector containing `style`

Reply:
Don't worry, I'm sure about it. Added the rule directly, thanks for opening the pull request in our repository!